client/core/helpers.go

```diff
@@ -141,28 +141,40 @@ func (ord *OrderReader) SettledPercent() string {
 }
 
 // FilledFrom is the sum filled in units of the outgoing asset.
-func (ord *OrderReader) FilledFrom() string {
+func (ord *OrderReader) FilledFrom(excludeCancels bool) string {
+	filter := filledFilter
+	if excludeCancels {
+		filter = filledNonCancelFilter
```

Comment:
Interestingly, the fix to use `filledNonCancelFilter` was applied long ago in https://github.com/decred/dcrdex/pull/816/commits/890ab64885907a2416e780e2f1e2d91bd89d894b (https://github.com/decred/dcrdex/pull/816) and then reverted in an apparent accidental regression in https://github.com/decred/dcrdex/commit/08a72d1cd085af6a89dc535ab03184efa6083ad9

Looking at the history of this issue, I don't think `false` would ever be the correct arg for `FilledFrom` or `FilledTo`.  Can you comment on the need to ever include cancels in the computation, other than retaining the same value in the CSV output in `(*Core).deleteOrderFn`?

Reply:
We also export orders using `(s *WebServer).handleExportOrders` and it's the same with  `(*Core).deleteOrderFn`. These are the only two places we'd want to include cancels to match internal behaviour.

Reply:
Thanks.  In both of those cases it's for the `FilledPercent`, and that percent in the exported data should be just like everywhere else -- based only on trade matches not cancel matches.

Reply:
> Unless I'm missing something else, the bool arg should be removed from all of these and the correct filter used consistently.

If this is what we want, then we shouldn't include cancel matches when setting `trade.Filled`?

Reply:
Yeah, I think that was a mistake.  But I think it's going to be hard to change now because for *internal* accounting purposes and other logic, we probably expect canceled orders to be 100% filled.  Again, it doesn't make sense to me, but lets just fix the display.

Reply:
Alright.

Reply:
e.g. 

```go
func (t *Trade) Remaining() uint64 {
	t.fillAmtMtx.RLock()
	defer t.fillAmtMtx.RUnlock()
	return t.Quantity - t.FillAmt
}
```

If we redefine `FillAmt`, lots will break.  Maybe we could separately check a canceled bool, etc.  It's going to be a can of worms.

Reply:
If it is worth it (which I don't think it is), you open an issue. I be glad to look into it.